bin/suitcss

```diff
@@ -63,9 +63,16 @@ program.parse(process.argv);
  * Settings.
  */
 
+function requireOrParse(configFile) {
+  var configPath = resolve(configFile);
+  var ext = path.extname(configPath);
+  var readFn = ext === '.js' ? require : fs.readJsonSync;
```

Comment:
I think we should lean in `require` where possible. It seems wasteful to send JSON files through [this extra module](https://github.com/jprichardson/node-jsonfile/blob/master/index.js#L50-L79) which basically does what `require` would for free.

Could we reinstate the regex or check for `json` as well?